### DISCUSSION TITLE
Welcome to watt Discussions!

### DISCUSSION THREAD

**Discussion by tnmendes** [tnmendes/watt#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by upriis** [comment]
GREAT new update v4

Thanks!

**Reply by tnmendes** [reply]
For some time that  I was thinking that the Watt deserved a redesign with everything I have learned at the level of user needs and so it happened :D
Thank you @upriis
